app/controllers/users/phones_controller.rb

```diff
@@ -13,11 +13,13 @@ class PhonesController < ApplicationController
     def add
       user_session[:phone_id] = nil
       @new_phone_form = NewPhoneForm.new(user: current_user, analytics: analytics)
+      track_add_phone_visit
```

Comment:
Minor: Should we inline the analytics call here, to reduce indirection since the method has just a single line, and for consistency with how we're calling analytics methods in the `#create` method?

```suggestion
      analytics.user_registration_add_phone_setup_visit
```